src/main/webapp/i18n/de/error.json

```
@@ -47,6 +47,32 @@
         "submissionPolicyToggleFailedPolicyNotExist": "Für diese Programmieraufgabe konnte die Abgaberichtlinie nicht (de)aktiviert werden, da die Programmieraufgabe keine Abgaberichtlinie hat. ",
         "submissionPolicyActiveNull": "Abgaberichtlinien müssen aktiviert oder deaktiviert sein. Dieser Status darf nicht null sein.",
         "submissionPolicyUpdateFailedPolicyNotExist": "Die Abgaberichtlinie konnte nicht gespeichert werden, da die Programmieraufgabe keine Abgaberichtlinie hat!",
-        "oneActivePlagiarismCheck": "Bitte warte bis die aktuelle Plagiatskontrolle in diesem Kurs abgeschlossen ist, bevor du eine weitere startest."
+        "oneActivePlagiarismCheck": "Bitte warte bis die aktuelle Plagiatskontrolle in diesem Kurs abgeschlossen ist, bevor du eine weitere startest.",
+        "internalServerError": "Interner Serverfehler",
+        "server.not.reachable": "Server ist nicht erreichbar",
+        "url.not.found": "Nicht gefunden",
+        "NotNull": "Feld {{fieldName}} kann nicht leer sein!",
```

```suggestion
        "notNull": "Feld {{fieldName}} darf nicht leer sein!",
```
Watch out, I also changed "NotNull" to "notNull"